blocks/api/serializer.js

```diff
@@ -32,16 +32,17 @@ export function getBlockDefaultClassname( blockName ) {
  *
  * @param  {Object} blockType  Block type
  * @param  {Object} attributes Block attributes
+ * @param  {?Array} children   Nested blocks
  * @return {string}            Save content
  */
-export function getSaveContent( blockType, attributes ) {
+export function getSaveContent( blockType, attributes, children = [] ) {
```

Comment:
What's the general thinking behind sometimes referring to `innerBlocks` and sometimes referring to `children`?

Reply:
>What's the general thinking behind sometimes referring to `innerBlocks` and sometimes referring to `children`?

We should probably align it. `innerBlocks` is the naming provided from the parser, but `children` seemed a friendlier / more familiar name. Maybe too familiar though, since it conflicts with the React built-in prop of the same name; could mislead a developer to think they could render it as an element set (which it is not).

Do you have a preference one way or the other?

Reply:
I'd lean toward `innerBlocks` for the reasons you cite, plus the fact that with `innerBlocks` we reinforce the idea of "blocks" as the central object. Also, the notion of containment behind _inner_ may be more fitting than that of descent behind _children_.